api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        try:
+            pointer = node.add_pointer(pointer_node, auth, save=True)
+            return pointer
+        except ValueError:
+            raise exceptions.NotFound('Pointer to node {} already in list'.format(pointer_node._id))
+        except AttributeError:
```

Comment:
`AttributeError` is raised because `pointer node` is `None`.  Alternatively, the `self.context['view'].get_node()` line raises a 404 if the node cannot be found.